db/models.py

```diff
@@ -1 +1,32 @@
 from django.db import models
+
+
+class Race(models.Model):
+    PLAYER_CHOICES = (
+        ("Elf", "Elf"),
+        ("Dwarf", "Dwarf"),
+        ("Human", "Human"),
+        ("Ork", "Ork"),
+    )
```

Comment:
If `PLAYER_CHOICES` is not used in the code and only takes up space, you can remove it.